database/migrations/2016_09_01_125807_add_name_location_url_to_friends.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddNameLocationUrlToFriends extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('friends', function (Blueprint $table) {
+            $table->string('name');
```

Comment:
Let's make all these fields nullable, since we don't know whether we'll get them back or not.